app/src/main/java/com/cornellappdev/resell/android/viewmodel/main/ChatViewModel.kt

```diff
@@ -49,9 +61,131 @@ class ChatViewModel @Inject constructor(
     }
 
     fun onBackPressed() {
-        applyMutation {
-            copy(currentChat = null)
-        }
         navController.popBackStack()
     }
+
+    fun onMeetingDetailsPressed() {
+        rootNavigationSheetRepository.showBottomSheet(
+            RootSheet.MeetingDetails(
+                callback = {
+
+                },
+                confirmString = "Cancel Meeting",
+                title = "Meeting Details",
+                closeString = "Close",
+                confirmColor = ResellTextButtonContainer.PRIMARY_RED
+            ) {
+                Spacer(Modifier.height(16.dp))
+
+                Text(
+                    text = "Meeting with Lia for Blue Pants confirmed for",
+                    style = Style.title4
+                )
+
+                Spacer(Modifier.height(16.dp))
+
+                Text(
+                    text = "Time",
+                    style = Style.title3
+                )
+                Text(
+                    text = "Friday, October 23 - 1:30-2:00 PM",
+                    style = Style.title4
+                )
+
+                Spacer(Modifier.height(16.dp))
+
+                Text(
+                    text = "For safety, make sure to meet up in a public space on campus",
+                    style = Style.title4
+                )
```

Comment:
Oh hm constructing and passing a `@Composable` in VM is kinda sus, we can keep for now but we'll prob want to refactor this in the future.

Reply:
Add a TODO before merging to replace w/ this data getting just passed as parameters, with this composable code in a `fun MeetingDetailsSheet()` instead